docker/DockerVerify.sh

```diff
@@ -26,18 +26,19 @@ die()
 }
 
 _positionals=()
-_arg_feature_path=('' )
 _arg_image_tag_prefix=
-_enable_test_processors=OFF
-
+_arg_tags_to_exclude=
+_arg_parallel_processes=3
```

Comment:
Can we increase the parallelism to further reduce runtimes, or are we running into some other bottleneck?

Reply:
Further increasing the parallelism caused instability on the ci machines without providing any real reduction in runtime. (locally with more ram and threads worked great)